<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsOfApp</class>
 <widget class="QWidget" name="settingsOfApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>301</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="security">
    <attribute name="title">
     <string>安全</string>
    </attribute>
    <widget class="QCheckBox" name="lock_CB">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>52</y>
       <width>181</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>将iPass上锁当它闲置</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>主密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mainPassword">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>8</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>更改主密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>自动锁闭</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>显示</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>剪贴板</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="lock_SB">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="lock_CoB">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>小时</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>分钟</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>秒</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="mask">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>122</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>隐藏敏感</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="clipBoard_CB">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>162</y>
       <width>121</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>后清除剪贴板</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="clipBoard_SB">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>192</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>秒</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="backup">
    <attribute name="title">
     <string>备份</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>导入导出日志</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>12</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>启用</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>381</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>305</y>
     <width>193</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
